tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers_test.ts

```diff
@@ -93,4 +98,398 @@ describe('Debugger reducers', () => {
       });
     });
   });
+
+  it('Overrides activeRunId on debuggerRunsLoaded', () => {
+    const state = createDebuggerState();
+    const nextState = reducers(
+      state,
+      actions.debuggerRunsLoaded({
+        runs: {
+          __default_debugger_run__: {
+            start_time: 222,
+          },
+        },
+      })
+    );
+    expect(nextState.activeRunId).toEqual('__default_debugger_run__');
+  });
+
+  it('Updates load state on numExecutionsRequested', () => {
+    const state = createDebuggerState({
+      runs: {
+        __default_debugger_run__: {
+          start_time: 111,
+        },
+      },
+      runsLoaded: {
+        state: DataLoadState.LOADED,
+        lastLoadedTimeInMs: 222,
+      },
+      activeRunId: '__default_debugger_run__',
+    });
+    const nextState = reducers(state, actions.numExecutionsRequested());
+    expect(nextState.executions.numExecutionsLoaded.state).toEqual(
+      DataLoadState.LOADING
+    );
+    expect(
+      nextState.executions.numExecutionsLoaded.lastLoadedTimeInMs
+    ).toBeNull();
+  });
+
+  it('Updates states correctly on numExecutionsLoaded', () => {
+    const state = createDebuggerState({
+      runs: {
+        __default_debugger_run__: {
+          start_time: 111,
+        },
+      },
+      runsLoaded: {
+        state: DataLoadState.LOADED,
+        lastLoadedTimeInMs: 222,
+      },
+      activeRunId: '__default_debugger_run__',
+    });
+    let nextState = reducers(state, actions.numExecutionsRequested());
+    const t0 = Date.now();
+    nextState = reducers(
+      nextState,
+      actions.numExecutionsLoaded({numExecutions: 1337})
+    );
+    expect(nextState.executions.numExecutionsLoaded.state).toEqual(
+      DataLoadState.LOADED
+    );
+    expect(
+      nextState.executions.numExecutionsLoaded.lastLoadedTimeInMs
+    ).toBeGreaterThanOrEqual(t0);
+    expect(nextState.executions.executionDigestsLoaded.numExecutions).toEqual(
+      1337
+    );
+  });
+
+  it('Updates states on executionDigestsRequested', () => {
+    const state = createDebuggerState({
+      runs: {
+        __default_debugger_run__: {
+          start_time: 111,
+        },
+      },
+      runsLoaded: {
+        state: DataLoadState.LOADED,
+        lastLoadedTimeInMs: 222,
+      },
+      activeRunId: '__default_debugger_run__',
+      executions: createDebuggerExecutionsState({
+        numExecutionsLoaded: {
+          state: DataLoadState.LOADED,
+          lastLoadedTimeInMs: 333,
+        },
+        executionDigestsLoaded: {
+          numExecutions: 1337,
+          pageLoadedSizes: {},
+          state: DataLoadState.NOT_LOADED,
+          lastLoadedTimeInMs: null,
+        },
+      }),
+    });
+    let nextState = reducers(state, actions.executionDigestsRequested());
```

Comment:
let -> const.

Reply:
Done. Did a pass to fix all oversights like this in this file.